.prettierignore

```diff
@@ -1,5 +1,5 @@
 src/theme/DocItem
-src/theme/DocPage
+src/theme/DocRoot
```

Comment:
DocPage has been replaced by DocRoot. The name is the only change that happened in Docusaurus v3.